ush/ufsda/yamltools.py

```diff
@@ -47,7 +47,12 @@ def parse_config(input_config_dict, template=None, clean=True):
     # compute common resolution variables
     if config_out.get('atm', True):
         config_out = atmanl_case(config_out)
+    elif config_out.get('land', True):
+        config_out = landanl_case(config_out)
+    else:
+        raise KeyError("Neither $(atm) nor ${land} defined")
```

Comment:
one is () and the other is {}, I don't care which for this error print but make it consistent

Reply:
Made change for keeping the consistency.